js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -51,6 +86,18 @@ const ConnectedGoogleComboAccountCard = () => {
 		( editMode && hasExistingGoogleAdsAccounts ) ||
 		( ! isConnected && hasExistingGoogleAdsAccounts );
 
+	// Show the spinner if there's an account creation in progress and account should not be claimed.
+	// If we are not showing the ConnectMC screen, for e.g when we are creating the first account,
+	// then show the spinner in the Google combo card while the Ads account is being claimed.
+	const showSpinner =
+		( Boolean( creatingWhich ) && ! shouldClaimGoogleAdsAccount ) ||
+		( ! showConnectAds && finalizeAdsAccountCreation );
+
+	const showConversionMeasurementNotice =
+		( googleAdsAccount?.status === GOOGLE_ADS_ACCOUNT_STATUS.CONNECTED ||
+			googleAdsAccount?.step === 'link_merchant' ) &&
```

Comment:
As mentioned in https://github.com/woocommerce/google-listings-and-ads/pull/2644#discussion_r1833445209:

> The only case in the old component where the action will NOT be shown is if an account is set up resulting in the 'billing' step being returned. I've confirmed this by creating a new account during the secondary Create Ads setup flow, which I believe to be an oversight.

Perhaps this condition should be taken into account when `step` is `'billing'`.

https://github.com/user-attachments/assets/4b284b1a-eabf-455a-831e-27ca56afce07

Reply:
I'm not sure I fully understand the suggestion here, but in order to account for both cases (when the step is either 'link_merchant' or 'billing', I've updated the logic used by both components in [ac3eef9](https://github.com/woocommerce/google-listings-and-ads/pull/2644/commits/ac3eef9d1113683baa76f27671c9508ddec20486) to use a common utility function that includes the 'billing' case.